PCbuild/pyproject-clangcl.props

```
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <__PyprojectClangCl_Props_Imported>true</__PyprojectClangCl_Props_Imported>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <_profrawFiles Include="$(OutDir)instrumented\$(TargetName)_*.profraw" />
+  </ItemGroup>
+
+  <Target Name="EnsureClangProfileData" BeforeTargets="PrepareForBuild"
+          Condition="'$(SupportPGO)' and $(Configuration) == 'PGUpdate'">
+    <Error Text="PGO run did not succeed (no $(TargetName)_*.profraw files) and there is no data to merge"
+           Condition="$(RequireProfileData) == 'true' and @(_profrawFiles) == ''" />
+  </Target>
+
+  <Target Name="MergeClangProfileData" BeforeTargets="PrepareForBuild"
+          Condition="'$(SupportPGO)' and $(Configuration) == 'PGUpdate'"
+          Inputs="@(_profrawFiles)"
+          Outputs="@(_profrawFiles->'$(OutDir)instrumented\profdata.profdata')">
```

```suggestion
          Outputs="$(OutDir)instrumented\profdata.profdata">
```

I'm 99% sure this should work (different numbers of inputs vs. outputs is fine), but if you tried it and it didn't then no worries.

---

Haven't tried before but can confirm your suggestion works!